src/BlogDetails.js

```diff
@@ -0,0 +1,38 @@
+import { useParams } from "react-router";
+import useFetch from './useFetch';
+import { useHistory } from 'react-router-dom';
+const BlogDetails = () => {
+
+    // Now we want to access whatever the id of blog on url router
+    // we put what we named our route paramters in App.js
+    const { id } = useParams();
+    // Reusing our custom hook
+    const { data , isPending, error} = useFetch('http://localhost:8000/blogs/' + id);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/7bel6/b-blog/pullRequest?prid=7919486)
```suggestion
    const { data , isPending, error} = useFetch("http://localhost:8000/blogs/" + id);
```